pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -189,6 +189,7 @@ def __init__(self, goal=None, planning_group=_DEFAULT_PLANNING_GROUP, target_lin
         self._relative = relative
 
         self._reference_frame = reference_frame
+        self._robot = None
```

Comment:
Imho the robot object should neither be passed around nor set as a member object of cmds. Would it be feasible, to read the required properties in `cmd._get_sequence_request()` and only pass the current robot state and speed_override (as members of the cmd or as parameters doesn't matter for me). I'd suggest to discuss this refactoring seperately as it is out-of-scope of the current PR.

Reply:
agree, that this may be done in a more clean way. Should i remove it out of this pr completly and go back to passing it around douzend times or leave it as is?

Reply:
ok, let's keep it as is and defer refactoring to a follow-up PR.

Reply:
#251